### ISSUE TITLE
Java 11 iText causes Illegal reflective access warning

### ISSUE THREAD

**Issue by hvbtup** [eclipse/birt#793]
I found this on stderr when I tested the new POJO runtime.

WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.lowagie.text.pdf.MappedRandomAccessFile$1 (file:/E://....birt-runtime-4.9.0-20220126/ReportEngine/lib/com.lowagie.text_2.1.7.v201004222200.jar) to method java.nio.DirectByteBuffer.cleaner()
...

Note: This is not a new issue - it is merely because BIRT still uses the ancient iText 2.1.7 and Java has changed over the years.

It could mean that BIRT PDF generation does not work with Java 17 or newer (did not test that).

Claes is working on replacing iText with OpenPDF (see https://github.com/eclipse/birt/pull/780), so hopefully this issue will be solved once that is done.

**Comment by hvbtup** [comment]
Sorry, this is already described in #767
